youtube_dl/extractor/svt.py

```diff
@@ -360,11 +364,26 @@ def _real_extract(self, url):
 
         webpage = self._download_webpage(url, playlist_id)
 
+        redux_state = self._parse_json(
+            self._search_regex(
+                r'\.\s*reduxState=\s*({.*})',
+                webpage,
+                'reduxState'),
+            None,
+            transform_source=js_to_json)
+
+        media = try_get(redux_state, lambda x: x['componentState']['modal']['media']['article']['media'], list) or []
```

Comment:
Inline.

Reply:
Could you please clarify what you mean by *inline*?

Reply:
Upon further investigation of the *coding conventions*, I assume you mean that the *media* variable should be disposed of, is that correct?